lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -6126,6 +6202,11 @@ void SharedEmitterState::gatherFiles(bool separateModules) {
     // Separate the operation into dedicated output file, or emit into the
     // root file, or replicate in all output files.
     TypeSwitch<Operation *>(&op)
+        .Case<emit::FileOp, emit::FileListOp>([&](auto file) {
+          // Emit file ops to their respective files.
+          symbolCache.addDefinition(file.getSymNameAttr(), file);
```

Comment:
It seems weird to add this symbol to the symbolCache as I always thought that was more of a mechanism for generating unique names as opposed to something that is used for symbol lookup. Is this the right way to do this?